pkg/kwok/metrics/cel/evaluate.go

```diff
@@ -88,12 +111,14 @@ func (e *Environment) init() error {
 	if e.conf.Now != nil {
 		funcs["now"] = append(funcs["now"], e.conf.Now)
 	} else {
-		funcs["now"] = append(funcs["now"], time.Now)
+		funcs["now"] = append(funcs["now"], timeNow)
 	}
 
-	unixSecond := func(t time.Time) int64 {
-		return t.Unix()
-	}
+	funcs["rand.double"] = append(funcs["rand.double"], mathRand)
+
+	methods["sinceSecond"] = append(methods["sinceSecond"], sinceSecond[*corev1.Node], sinceSecond[*corev1.Pod])
```

Comment:
I believe that the function names registered with cel, such as `rand.double`, `sinceSecond`, and `unixSecond`, should maintain a consistent style for the convenience of users.

Reply:
Can you give an example?

Reply:
e.g. `rand.double`, `time. sinceSecond`,`time.unixSecond `

Reply:
emmm, in the expressions node, pod, container means Object, and math, time means package, So `pod.sinceSecond()` and `sinceSecond(pod)` might have to be registered as `pod.(time.sinceSecond)()` and `time. SinceSecond(pod)` doesn't feel right to me, and I feel even worse if the method and func are registered under different names, so I'd remove the package layer.